content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -152,7 +152,8 @@ different Kubernetes components.
 | `VolumeSubpathEnvExpansion` | `true` | Beta | 1.15 | |
 | `VolumeSnapshotDataSource` | `false` | Alpha | 1.12 | - |
 | `WatchBookmark` | `false` | Alpha | 1.15 | 1.15 |
-| `WatchBookmark` | `true` | Beta | 1.16 | |
+| `WatchBookmark` | `true` | Beta | 1.16 | 1.16 |
+| `WatchBookmark` | `true` | GA | 1.17 | - |
```

Comment:
hello @wojtek-t . Recently, a second table was added for features that have reached GA. If `WatchBookmark` is reaching GA (1.17), then the three `Watch` entries belong in the second table. @tengqm , should be able to confirm.

Reply:
done